### PR TITLE
Remove redundant uses of 'document' in our code

### PR DESCRIPTION
**Pull request by MuriloDalRi**
https://trello.com/c/YpJhGyqy/548-remove-redundant-uses-of-document-in-our-code

Some of our controllers and services unnecessarily use the word 'document' to describe themselves e.g. 'DocumentImagesController' and 'DocumentUnpublishingService'. We should remove these uses to make our naming more consistent.

### CODE REVIEW

File: app/services/unpublishing_service.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class DocumentUnpublishingService
+class UnpublishingService
```
Comment (benthorner) [inline app/services/unpublishing_service.rb]: This one should probably be 'UnpublishService' for consistency.

Comment (MuriloDalRi) [reply app/services/unpublishing_service.rb]: Oops missed a few letters here, well spotted!

**Comment by benthorner** [comment]
Hmm...I think you missed renaming the unpublishing specs. Could you do another PR for that?

**Comment by MuriloDalRi** [comment]
It's just the filename, right? I'll open a PR

**Pull request merged**